src/js/components/WaitForSettings/WaitForSettings.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { isActiveZoneOnCloudflare } from '../../selectors/activeZone';
+import Loading from 'cf-component-loading';
+import Text from 'cf-component-text';
+import { FormattedMessage, injectIntl } from 'react-intl';
+
+class WaitForSettings extends Component {
+
+  render() {
+    let activeZone = this.props.activeZone;
+    let analytics = (typeof this.props.analytics != 'undefined' ? this.props.analyics : true);
+    let settings = (typeof this.props.settings != 'undefined' ? this.props.settings : true);
```

Comment:
I'm confused by your comment because putting `undefined` in single quotes makes it a string and [`typeof` returns a string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof).

The selectors we use in HomePage, MoreSettingsPage, and AnalyticsPage return `false` if the various settings aren't found.  If the prop is `undefined` that means it wasn't passed in and we don't need to check it so we set it to `true`.